app/scripts/modules/azure/loadBalancer/loadBalancer.transformer.js

```diff
@@ -146,24 +146,32 @@ module.exports = angular.module('spinnaker.azure.loadBalancer.transformer', [
         detail: 'frontend',
         credentials: defaultCredentials,
         region: defaultRegion,
-        vpcId: null,
-        healthCheckProtocol: 'HTTP',
-        healthCheckPort: 7001,
-        healthCheckPath: '/healthcheck',
-        healthTimeout: 5,
-        healthInterval: 10,
-        healthyThreshold: 10,
-        unhealthyThreshold: 2,
-        regionZones: [],
+        cloudProvider: 'azure',
+        vnet: null,
+        probes: [
+          {
+            probeName: '',
+            probeProtocol: 'HTTP',
+            probePort: 7001,
+            probePath: '/healthcheck',
+            probeInterval: 10,
+            unhealthyThreshold: 2
+          }
+        ],
         securityGroups: [],
-        listeners: [
+        loadBalancingRules: [
           {
-            internalProtocol: 'HTTP',
-            internalPort: 7001,
-            externalProtocol: 'HTTP',
-            externalPort: 80
+            ruleName: '',
+            protocol: 'TCP',
+            externalPort: 80,
+            backendPort: 8080,
+            probeName: '',
+            persistence: 'None',
+            idleTimeout: 4,
           }
-        ]
+        ],
+        name: '',
+        user: 'anonymous',
```

Comment:
I think these two fields can be stripped out - they're not hurting anything, but they'll be overwritten by the `taskExecutor`.